resources/RegexGrammar.pp

```diff
@@ -177,10 +179,14 @@
 
 #class:
     (
-        ::negative_class_:: #negativeclass
+        ::negative_class_fc_:: #negativeclass
```

Comment:
```suggestion
        ::negative_class_fc_:: #negative first-char class
```

Reply:
why? - `#negativeclass` should be there 2x the same as we want 2x the same AST

Reply:
Yeah this is not a comment, this is a marker so it appears as `negativeclass` in the AST

Reply:
But adding a comment with // on the %token above could make sense

Reply:
> But adding a comment with // on the %token above could make sense

already done in https://github.com/phpstan/phpstan-src/pull/3488#discussion_r1778117255